pinc/ProjectSearchResults.inc

```diff
@@ -665,8 +671,7 @@ class ProjectSearchResults
         echo "</tr>\n";
 
         while ($project_assoc = mysqli_fetch_assoc($result)) {
-            $project = new Project($project_assoc);
-            $projectid = $project->projectid;
+            $project = new ProjectSearchResultEntry($project_assoc);
```

Comment:
Nit: Adding this new type is reasonable, though I have a hard finding if/where `$hold` is set...

Also the name is a bit confusing (it's not a `$project`, it's a `$project_result_entry`) so it may be worth renaming it so we are clear on what we are manipulating in this function.

Reply:
`hold` is loaded into `$project_assoc` array via the SQL at https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/ProjectSearchResults.inc#L615 and set as variable in the `Project` [constructor](https://github.com/DistributedProofreaders/dproofreaders/blob/master/pinc/Project.inc#L240) from that associative array.

I'm going to add a comment about renaming this variable when we add type definitions to this file as it will change a ton of class function definitions as part of it and we might as well do it all at once.

Reply:
Thanks for the context on where `hold` comes from and sounds great!